### ISSUE TITLE
JSON-LD: literals should not have URIs

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#3847]
JSON-LD (and RDF in general) does not assign URIs to literal values.  So a label that has a data type of rdfs:Literal or xsd:string should be just a string, not a JSON object.

Currently a literal gets exported as

```
"label": {"id": "http://localhost:8000/tile/14d2e5cb-6fbf-4453-9551-61e6b0139b9f/node/d6392fe3-9a7f-11e8-996e-a4d18cec433a", "type": "xsd:string", "value": "test"},
```

And it should be just 

```
"label": "test"
```

**Comment by apeters** [comment]
currently all node values resolve to Literals of type RDF.value

eg: 
```
"http://www.w3.org/1999/02/22-rdf-syntax-ns#value": "{\"features\": [{\"geometry\": {\"coordinates\": [-0.5361532405497371, 53.23412529821837], \"type\": \"Point\"}, \"id\": \"d69be94711762a77902def06a436c7af\", \"properties\": {}, \"type\": \"Feature\"}], \"type\": \"FeatureCollection\"}"
```

**Comment by azaroth42** [comment]
Right, which is not correct. Values should resolve to values, not triples.  `rdf:value` isn't a class, it's a property that a class has. The class would be rdf:Literal (or xsd:string, or xsd:integer or ...) ... which is also unnecessary to represent in the data in JSON-LD.

**Comment by apeters** [comment]
We could export a literal that way you propose.

```
"label": "test"
```

We could just never PUT it with any confidence.  We would need a tileid for that.

**Comment by azaroth42** [comment]
Can you elaborate as to the issue with PUT/POST? Is it the same as the branch selection issue?

**Comment by apeters** [comment]
Right now, if you were to PUT this fragment back to the API, I would know what tileid and node to update
```
"label": {
    "id": "http://localhost:8000/tile/14d2e5cb-6fbf-4453-9551-61e6b0139b9f/node/d6392fe3-9a7f-11e8-996e-a4d18cec433a",
    "type": "xsd:string",
    "value": "test"
}
```

With this fragment, if it were PUT back, I wouldn't be able to know what tile was being updated.
```
"label": "test"
```

**Comment by azaroth42** [comment]
Is it necessary to update individual tiles like this?

Instead, it seems easier if it would just update the `label` of the resource to have a single value of "test". If you wanted to add a label of "test", to an existing label of "foo", then you would PUT `"label": ["test", "foo"]` destroying the existing "foo" tile and creating new ones.
